browser/ui/views/location_bar/brave_location_bar_view.cc

```diff
@@ -78,6 +78,11 @@ absl::optional<BraveColorIds> GetFocusRingColor(Profile* profile) {
 
 }  // namespace
 
+bool BraveLocationBarView::ShowPageActions() const {
+  // Do not show actions whilst omnibar is open or url is being edited
+  return !ShouldHidePageActionIcons() || omnibox_view_->GetText().empty();
```

Comment:
How about syncing with upstream's behavior?
Upstream's icon is also hidden when omnibox is in editing mode and it's empty.
but, our icon is visible because of this `omnibox_view_->GetText().empty()` condition with below steps.

1. Visit https://theatlantic.com/
2. The RSS icon should show up.
3. Click omnibox and delete all
4. Omnibox is still in editing mode but empty text
5. Brave's icon gets visible but upstream's icon is still not visible

Reply:
I'm on board with this, just didn't want to change the existing behavior. Updating it now.

Reply:
FYI, to avoid `ios` CI error, we need latest master :)